.gitignore

```diff
@@ -8,6 +8,8 @@ var
 .vitepress
 __pycache__
 packages/php-wasm/node/src/test/__test*
+packages/docs/site/src/model.json
```

Comment:
@adamziel why is this file excluded? Docs builds are currently failing and I'm considering including it.

Reply:
The file is dynamically generated